source/installation/restore-minio.rst

```diff
@@ -42,14 +42,18 @@ and links to procedures for recovering from each:
 .. admonition:: MinIO Professional Support
    :class: note
 
-   |subnet| provides 24/7 direct-to-engineering support for MinIO 
-   deployments. Consider leveraging SUBNET to ensure smooth and reliable
-   operation of MinIO deployments, including root-cause analysis, 
-   health diagnostics, and architecture reviews.
+   |subnet| provides 24/7 direct-to-engineering support for MinIO deployments.
+   SUBNET is a key component for ensuring smooth and reliable operation of MinIO
```

Comment:
"and reliable operation of production MinIO deployments" ?

Reply:
Done